Bugzilla/Template.pm

```diff
@@ -888,7 +888,9 @@ sub create {
             },
 
             json_encode => sub {
-                return encode_json($_[0]);
+                my $str = encode_json($_[0]);
```

Comment:
I'm sure this works, but using ::is_utf8() is wrong (it doesn't mean anything for the purposes of encoding).

Reply:
@kyoshino:
I suggest this change: https://gist.github.com/dylanwh/001e58d22765d4b500eac97e504167f4

Mojo::JSON is defined as returning bytes (not characters) and Encode::decode will take bytes and turn them into characters (assuming the bytes are utf8). Anything that is invalid (e.g. not utf8) will be replaced with the replacement character.

Reply:
I copied this from somewhere else. Without this, `js_BUGZILLA.string.InlineCommentEditor` will contain garbage characters.  Let’s use Mojo then 😺